private/js/webpack.config.js

```diff
@@ -1,7 +1,8 @@
 const argv = require('minimist')(process.argv.slice(2));
 const plugins = [];
 const webpack = require('webpack');
-const path = require('path');
+const path = require('path');1
```

Comment:
```suggestion
const path = require('path');
```

I assume the `1` is a typo here?